### PR TITLE
Update pull-request-template.md

### PR DESCRIPTION
**Pull request by mPokornyETM**
# Update pull request template

## Basic information

> Pull request templates allow your organizations to have a default text when you create a pull request on GitHub. It is quite useful to make sure to follow a standard process for every pull request and to have a to-do list for the author to check before requesting a review

And this is it. A good template improve the PR process and helps to find issues by review.

### Testing done

No code changes. This change can be tested by next PR first.

### Proposed upgrade guidelines

N/A

### Localizations

- [x] English
- ~~[ ] German~~

### Submitter checklist

- ~~[ ] The Github issue, if it exists, is well-described.~~
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood.
  - The changelog generator for plugins uses the **pull request title as the changelog entry**.
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during the upgrade.
- [x] There is automated testing or an explanation that explains why this change has no tests.
- ~~[ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.~~
- ~~[ ] Any localizations are transferred to /msg/ files.~~

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] Changelog entries in the **pull request title** and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- ~~[ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](...)).~~
- ~~[ ] cpp and ctrl code changes are tested by automated test.~~
- ~~[ ] WinCC OA guidelines for C++ and Control(++) coding have been met.~~
- ~~[ ] Result of pipeline build proving error/warning free code.~~
- ~~[ ] Result of automatic tests proving regression free.~~

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 1]
```diff
@@ -10,11 +10,81 @@ assignees: ''
...
+<!-- TBD, maybe done in GitHub rules
+- [ ] There is at least one (1) approval for the pull request and no outstanding requests for change.
+- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
+-->
+- [ ] Changelog entries in the **pull request title** and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
+<!-- TBD, maybe done in GitHub rules
+- [ ] Proper changelog labels are set so that the changelog can be generated automatically. See also [release-drafter-labels](...).
+-->
+- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](...)).
+- [ ] cpp and ctrl code changes are tested by automated test.
```
Comment (dhoegerlETM) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:87]:
```suggestion
- [ ] C++ and Control(++) code changes are tested by automated test.
```
Use wording as in next line.

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md [hunk 2]
```diff
@@ -10,11 +10,81 @@ assignees: ''
...
+
+<!-- TBD, maybe done in GitHub rules
+- [ ] There is at least one (1) approval for the pull request and no outstanding requests for change.
+- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
+-->
+- [ ] Changelog entries in the **pull request title** and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
+<!-- TBD, maybe done in GitHub rules
+- [ ] Proper changelog labels are set so that the changelog can be generated automatically. See also [release-drafter-labels](...).
+-->
+- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](...)).
```
Comment (dhoegerlETM) [inline .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:86]: We don't have this label _upgrade-guide-needed_ yet, so please mention _documentation_ instead or explain, why we would need another label.

**Review by dhoegerlETM — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md (hunk 2 above)
Comment (mPokornyETM) [reply .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:86]: Have we some labels? A good example is here: 
https://github.com/jenkinsci/.github/blob/ce466227c534c42820a597cb8e9cac2f2334920a/.github/release-drafter.yml#L9-L50
I will remove (comment out) the line from Template

**Review by dhoegerlETM — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull-request-template.md (hunk 2 above)
Comment (dhoegerlETM) [reply .github/PULL_REQUEST_TEMPLATE/pull-request-template.md:91]: See the current list here:
https://github.com/siemens/CtrlppCheck/issues/labels

**Pull request merged**